docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,45 @@
+
+= Scheduled Outages
+
+Scheduled Outages allow for suppressing notifications and pausing Status Polling, Threshold Checking, and Data Collection in {page-component-title}.
+{page-component-title} supports many configurable options for when scheduled outages need to occur.
+
+Scheduled Outages is located under Admin -> Service Monitoring -> Configure Scheduled Outages
+
+== Components of a Scheduled Outage
```

Comment:
I would suggest combining this section with the "Editing Outage" section to make this a process on creating a scheduled outage. We can also include a section on editing an existing outage if we want, but its contents would largely be the same.

```suggestion
== Create a scheduled outage

```

Reply:
Might this work better as a table listing out all of these items and what the possible configurations are?
Something like the "Configuration and use" section at the [NtpMonitor service monitors page](https://docs.opennms.com/horizon/31/reference/service-assurance/monitors/NtpMonitor.html#configuration-and-use)?